lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -185,7 +185,7 @@ private static function get_inner_blocks_html( $attributes, $inner_blocks ) {
 	private static function get_inner_blocks_from_navigation_post( $attributes ) {
 		$navigation_post = get_post( $attributes['ref'] );
 		if ( ! isset( $navigation_post ) ) {
-			return '';
```

Comment:
These functions should maybe return an empty `WP_Block_List` to be consistent. 🤔